### ISSUE TITLE
notice the diffusers version

### ISSUE THREAD

**Issue by qinghew** [PRIS-CV/Omnieraser#7]
pip install diffusers==0.33.0
or
pip install git+https://github.com/huggingface/diffusers.git@464374fb87610c53b2cf81e08d3df628fada3ce4#egg=diffusers 

in addition, if meet: Error creating pre-trained FluxPipeline with diffusers "Cannot instantiate this tokenizer from a slow version"
`pip install sentencepiece`
will address the problem

**Comment by qinghew** [comment]
i provide a refined controlnet version for the image path problem.

`import torch
import os
import cv2
from PIL import Image
from tqdm import tqdm
import numpy as np
from diffusers.utils import load_image, check_min_version
from controlnet_flux import FluxControlNetModel
from transformer_flux import FluxTransformer2DModel
from pipeline_flux_controlnet_removal import FluxControlNetInpaintingPipeline

check_min_version("0.30.2")

name = '0bce8e90-10f1-442e-8330-2917fc7fa486.png'
images_path = f'example/image/'
masks_path = f'example/mask/'
outputs_path = 'example/output/'
os.makedirs(outputs_path, exist_ok=True)

# Build pipeline
controlnet = FluxControlNetModel.from_pretrained("FLUX.1-dev-Controlnet-Inpainting-Beta", torch_dtype=torch.bfloat16)
transformer = FluxTransformer2DModel.from_pretrained(
        "/ytech_m2v2_hdd/baijianhong/flux/ckpt_dev", subfolder='transformer', torch_dtype=torch.bfloat16
    )
pipe = FluxControlNetInpaintingPipeline.from_pretrained(
    flux/ckpt_dev",
    controlnet=controlnet,
    transformer=transformer,
    torch_dtype=torch.bfloat16
).to("cuda")

pipe.load_lora_weights('Omnieraser_Controlnet_version', weight_name="controlnet_flux_pytorch_lora_weights.safetensors")
pipe.transformer.to(torch.bfloat16)
pipe.controlnet.to(torch.bfloat16)


image_path = os.path.join(images_path, name)
mask_path = os.path.join(masks_path, name)
output_path = os.path.join(outputs_path, name)
prompt='There is nothing here.'
# Load image and mask
size = (1024, 1024)
image = load_image(image_path).convert("RGB").resize(size)
#mask = load_image(mask_path).convert("RGB").resize(size)
mask = cv2.imread(mask_path, cv2.IMREAD_GRAYSCALE)
mask = Image.fromarray(mask.astype(np.uint8).repeat(3,-1)).convert("RGB")
mask = mask.resize((1024, 1024))
generator = torch.Generator(device="cuda").manual_seed(24)

# Inpaint
result = pipe(
    prompt=prompt,
    height=size[1],
    width=size[0],
    control_image=image,
    control_mask=mask,
    num_inference_steps=28,
    true_guidance_scale=1.0,
    guidance_scale=3.5,
    generator=generator,
    controlnet_conditioning_scale=0.9
).images[0]
result.save(output_path)

`

**Comment by weathon** [comment]
thanks
